src/components/chart/specific-chart-options.js

```diff
@@ -53,18 +53,37 @@ class SpecificChartOptions {
 
                         if (type === 'line') {
                             currentChart.series.forEach((series) => {
-                                const points = series.points;
-                                if (points && points.length > 0) {
-                                    // Show only the last point marker
-                                    const lastPoint = points[points.length - 1];
-                                    lastPoint.update(
+                                if (series.options.marker.enabled === 'true') {
```

Comment:
Hey @precious-onyenaucheya-ons this isn't behaving as expected - if it is OK with you I will push a change to fix this.

Reply:
I have now updated it as follows:

- If markers are enabled, all the dots show
- If markers are disabled, only the last dot shows

In a later PR we'll update the marker styles for each line, as per https://jira.ons.gov.uk/browse/CCB-38